.gitignore

```diff
@@ -1,7 +1,7 @@
 # Global build-related files
 
-node_modules # @oss-only
-**/node_modules # @oss-only
+node_modules
+**/node_modules
 
```

Comment:
Ah, thanks for catching this but IIRC, this is needed internally. 
I think I expected `# @oss-only` to be stripped by the Meta internal OSS tooling but it didn't work that way...
I won't go into details about why it was needed publicly. Happy to chat with anyone still at Meta if interested.

The workaround in OSS-land would be to ignore this absolute path in the [git global ignore config `~/.config/git/ignore`](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)